packages/core/src/stylable-resolver.ts

```diff
@@ -340,9 +350,10 @@ export class StylableResolver {
             }
             resolvedSymbols.mainNamespace[name] = deepResolved.symbol._kind;
         }
+        // ToDo: unify resolve types
```

Comment:
?

Reply:
got it.... `[[CSSKeyframes, 'keyframes'], [CSSLayer, 'layer']].forEach` idea. I think we should do it